content/en/blog/_posts/2021-06-18-Announcing-Kubernetes-Community-Group-Annual-Reports.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: blog
+title: "Announcing Kubernetes Community Group Annual Reports"
+description: >
+  Introducing brand new Kubernetes Community Group Annual Reports for
+  Special Interest Groups and Working Groups.
+date: 2021-06-23T10:00:00-08:00
+slug: Announcing-Kubernetes-Community-Group-Annual-Reports
+---
+
+**Authors:** Divya Mohan
+
+Given the growth and scale of the Kubernetes project, the existing reporting mechanisms were proving to be inadequate and challenging.
+Kubernetes is a large open source project. With over 100000 commits just to the main k/kubernetes repository, hundreds of other code
+repositories in the project, and thousands of contributors, there's a lot going on. In fact, there are 37 contributor groups at the time of
+writing. We also value all forms of contribution and not just code changes.
+
+With that context in mind, the challenge of reporting on all this activity was a call to action for exploring better options. Therefore
+inspired by the Apache Software Foundation’s [open guide to PMC Reporting](https://www.apache.org/foundation/board/reporting) and the
+[CNCF project Annual Reporting](https://www.cncf.io/cncf-annual-report-2020/), the Kubernetes project is proud to announce the 
+**Kubernetes Community Group Annual Reports for Special Interest Groups (SIGs) and Working Groups (WGs)**. In its flagship edition, 
+the [2020 Summary report](https://github.com/kubernetes/steering/blob/master/reports/summary-2020.md) focuses on bettering the 
+Kubernetes ecosystem by assessing and promoting the healthiness of the groups within the upstream community. 
+
+Previously, the mechanisms for the Kubernetes project overall to report on groups and their activitie were 
+(devstats)[https://k8s.devstats.cncf.io/], GitHub data, issues, to measure the healthiness of a given UG/WG/SIG/Committee. As a 
+project spanning several diverse communities, it was essential to have something that captured the human side of things. With 50,000+
+contributors, it’s easy to assume that the project has enough help and this report surfaces more information than /help-wanted and 
+/good-first-issue for end users. This is how we sustain the project. Paraphrasing one of the Steering Committee members, 
```

Comment:
```suggestion
[devstats](https://k8s.devstats.cncf.io/), GitHub data, issues, and their quarterly Community Meeting readout to gauge the healthiness of a given Community Group. As a 
project spanning several diverse communities, it was essential to have something that captured the human side of things. With 50,000+
contributors, it’s easy to assume that the project has enough help, but that is far from the case, with the report
surfacing more information than what can be found via [`/help-wanted`](http://go.k8s.io/help-wanted)
and [`/good-first-issue`](http://go.k8s.io/good-first-issue) labeled issued.
This is how we sustain the project. Paraphrasing one of the Steering Committee members, 
```
We only mention SIG/WG earlier, switching to community groups covers everything. Also linked to our help wanted/gfi links in case folks do want to go that route heh